diff --git a/autoload/riv.vim b/autoload/riv.vim
index ce1632f..dd7a30a 100644
--- a/autoload/riv.vim
+++ b/autoload/riv.vim
@@ -424,9 +424,9 @@ fun! riv#init() "{{{
     call riv#load_conf()
     call riv#load_aug()
     sil! let _t = clickable#highlight#init()
-    if !exists("_t")
-        echom "[RIV] clickable.vim is needed for link. Add it to your bundle"
-    endif
+    "if !exists("_t")
+    "    echom "[RIV] clickable.vim is needed for link. Add it to your bundle"
+    "endif
 endfun "}}}
 
 fun! riv#buf_load_aug() "{{{
